src/main/java/io/github/alathra/alathraskills/gui/GuiItemHelper.java

```diff
@@ -0,0 +1,87 @@
+package io.github.alathra.alathraskills.gui;
+
+import com.github.milkdrinkers.colorparser.ColorParser;
+import io.github.alathra.alathraskills.AlathraSkills;
+import io.github.alathra.alathraskills.api.SkillsPlayer;
+import io.github.alathra.alathraskills.utility.Cfg;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextDecoration;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.List;
+
+public class GuiItemHelper {
+
+    public static ItemStack borderItem;
+    public static ItemStack backItem;
+    public static ItemStack exitItem;
+    public static ItemStack confirmItem;
+    public static ItemStack denyItem;
+
+
+    public static void loadItemStacks() {
```

Comment:
I'm assuming `GuiItemHelper#loadItemStacks` is called sometime during startup. Since these item objects are cached, they will not properly apply config changes in the future if translations are added for the strings. 

I'd also recommend you cleanup this class and have the item objects available immediately when the class is initialized (*since the variable are supposed to be non-nullable *) by using a **[static initialization block](https://www.baeldung.com/java-static-instance-initializer-blocks)**, which will allow you to run the code in `GuiItemHelper#loadItemStacks`. Except the items will be guaranteed to exist on this class.

Reply:
When it comes to the potential config changes affecting the strings, it's definitely something I want to look into - but right now it's not really worth putting that much time into imo. 

I'll look at using static initialization for this, thanks for the suggestion.